### ISSUE TITLE
Krohnkite not functioning on KDE6

### ISSUE THREAD

**Issue by orange-toothbrush** [esjeon/krohnkite#218]
**Symptom**
Upon upgrading to KDE6, Krohnkite gets removed from KWin scripts. I tried to install from file again, and the .kwinscript file would not upload. KDE says it is invalid.

**How to Reproduce**
1. Upgrade to KDE
2. Open KWin Scripts settings, see that Krohnkite is gone

**Expected behavior**
Krohnkite continues to work with KDE6

**Environment**
OS: Artix Linux x86_64 
DE: Plasma 6.0.1 
WM: kwin 
WM Theme: Breeze

**Comment by RudraPS97** [comment]
Did you try Polonium? It's based on bismuth and I think bismuth was based on Krohnkite. ~~I cannot get it to work for whatever reason.~~
I got it to work. It's very basic for now. I hope Krohnkite gets updated to KDE6.
There are two forks of Krohnkite with updated code for plasma 5.27, but I can't get them to compile.

**Comment by orange-toothbrush** [comment]
Thanks @RudraPS97. I installed polonium and you're right. It's close, but not the same. Good enough for now.

**Comment by RudraPS97** [comment]
There is a fork of this project by anametologin. Seems to be working with plasma6.

**Comment by KBeDevel** [comment]
> There is a fork of this project by anametologin. Seems to be working with plasma6.

Yes, is currently published as 0.9.4 (15e916bcbf1e1dcdbba00a02917d82f146a948e5).

Seems stable for now, at least on Fedora 40 (w/ Plasma 6.0.5, FW 6.2.0 and Wayland).
